content/pages/core/discussion.md

```diff
@@ -86,7 +86,9 @@ but developers should be careful to transfer all the official decisions or usefu
 
 ## Slack
 
-The project's Slack channel is the **#lucene-dev** channel in the **the-asf** organization. Link: <https://the-asf.slack.com/messages/CE70MDPMF>
+- The project's Slack channel are the `#lucene-dev` and `#solr-dev` channels in the `the-asf` organization. These are primarily for developer discussions and not meant as support channels. Link: <https://the-asf.slack.com/messages/CE70MDPMF>
+- For Solr support, there is a (community maintained/unofficial) Slack organization that relays messages bi-directionally to/from the officially supported IRC channels. Link: https://s.apache.org/solr-slack
+- For relevance related discussions (Solr or other search engines), there's an unofficial Slack organization: https://opensourceconnections.com/slack
```

Comment:
This slack communities name is "Relevance Slack", the invite code just happens to be through OSC.   So how about "For relevance related discussions (Solr or other search engines), join Relevance Slack: http://opensourceconnections.com/slack"